docs/model_server_c_api.md

```diff
@@ -39,11 +39,10 @@ To process response, first you must check for inference error. If no error occur
 
 ## Preview limitations
 * Launching server in single model mode is not supported. You must use configuration file.
-* DAG pipelines cannot be used directly through C API. DAG inferences can be scheduled only by gRPC/HTTP endpoints.
-* There is no support for native file format (jpg/png) through C API.
+* There is no direct support for jpeg/png encoded input format through C API.
 * There are no server live, server ready, model ready, model metadata, metrics endpoints exposed through C API.
-* Inference scheduled through C API does not have inference success/failure, request time metrics counted.
```

Comment:
C-API counts some latency related metrics. It does not count `request_time` only.